packages/vite/src/node/config.ts

```diff
@@ -817,6 +817,13 @@ export async function loadConfigFromFile(
         isTS = true
       }
     }
+
+    if (!resolvedPath) {
+      const cjsConfigFile = path.resolve(configRoot, 'vite.config.cjs')
+      if (fs.existsSync(cjsConfigFile)) {
+        resolvedPath = cjsConfigFile
```

Comment:
This needs to set `isESM = false` since it would have been set to `true` in a `type: "module"` project